lambda/security-groups/src/main/scala/com/gu/hq/Lambda.scala

```diff
@@ -3,7 +3,7 @@ package com.gu.hq
 import com.amazonaws.regions.Regions
 import com.amazonaws.services.lambda.runtime.events.ConfigEvent
 import com.amazonaws.services.lambda.runtime.{Context, RequestHandler}
-import com.gu.hq.lambda.ConfigEventLogic
+import com.gu.hq.lambda.JsonParsing
```

Comment:
I don't see a corresponding change for the tests, have I missed something from a little computer? At least the test string label and probably the class, if not the tests themselves as well?

Reply:
Yes, the test was renamed to JsonParsingTest.scala.

Reply:
And, of course, the tests still pass.